core/pkg/service/flag-evaluation/connect_service.go

```diff
@@ -129,16 +130,28 @@ func (s *ConnectService) setupServer(svcConf service.Configuration) (net.Listene
 
 	path, handler := schemaConnectV1.NewServiceHandler(fes, append(svcConf.Options, marshalOpts)...)
 	mux.Handle(path, handler)
+	mux.Handle(grpchealth.NewHandler(grpchealth.NewStaticChecker(
```

Comment:
Standard implementation, recommended by buf/conntect.

You can test this locally with `grpcurl` (below command assumes you've downloaded the [health proto](https://github.com/grpc/grpc/blob/master/src/proto/grpc/health/v1/health.proto) to `~/Downloads`:

```bash
grpcurl -import-path '~/Downloads/'  -proto health.proto -plaintext localhost:8013 grpc.health.v1.Health/Check
```